mods/persistence/modules/mob/living/carbon/human/human.dm

```diff
@@ -20,13 +20,17 @@
 
 	if(ignore_persistent_spawn())
 		return
+
+	//#FIXME: This is kinda clunky and probably shouldn't be in the mob's code, or this early in the mob init.
+	//			Partially because its going to trigger move, onEnter, and etc events before init.
 	if(!loc) // We're loading into null-space because we were in an unsaved level or intentionally in limbo. Move them to the last valid spawn.
 		if(istype(home_spawn))
 			if(home_spawn.loc)
 				forceMove(get_turf(home_spawn)) // Welcome home!
 				return
 			else // Your bed is in nullspace with you!
 				QDEL_NULL(home_spawn)
+		message_staff("'[src]'(ckey:'[ckey]') loaded into nullspace, without a home_spawn set! Moving into closest valid location.")
```

Comment:
This can be left since it's good to get a heads up on it, but for reference this can occur without any bugs or anything. If someone was in a z-level that wasn't saved (ie a Deep Space z-level) and they don't have a home spawn, they'll intentionally be dropped back on Outreach.